server/models/Interview.mjs

```diff
@@ -0,0 +1,31 @@
+import mongoose from 'mongoose'
+
+const interviewSchema = new mongoose.Schema({
+  interview_id: { type: Number, required: true, unique: true },
+  user_id: { type: Number, ref: 'User', required: true },
+  application_id: { type: Number, ref: 'Job_Application', required: true },
+  interview_date: { type: Date },
+  interview_time: { type: String },
+  interview_location: { type: String },
+  interviewer_name: { type: String },
+  interview_notes: { type: String }
```

Comment:
For the schema field names, I went with camelCase since that's the convention for Javascript, but I don't think it matters. We should pick one though. On that topic, I had a hard time deciding between singular or plural for collection names (e.g., User vs Users), so I'm open to suggestions. Thinking about it more now, maybe plural would be better?

Reply:
Good callout, I like CamelCase better. I was able to have Github Copilot update all the fields in one swoop.

As far as plurality goes, when we're referring to these models, I think we should use singular naming, because the model represents a single object (i.e. one interviewer or one user). When we start referencing collections, that's when I think we should use plural since it refers to potentially multiple of those objects.

But, I am quite rusty on MongoDB, my most recent DB experience is the SQL world, so I may be missing something. We can discuss further on teams!

Reply:
I should've known Copilot would make that easy. Good call!